tests/admin_views/tests.py

```diff
@@ -807,6 +807,36 @@ def test_limited_filter(self):
             msg_prefix="Changelist filter not correctly limited by limit_choices_to",
         )
 
+    def test_change_list_facet_toggle(self):
+        # Toggle visible when show_facet is the default of admin.ShowFacets.ALLOW
```

Comment:
Wrap comments at 79 chars
```suggestion
        # Toggle is visible when show_facet is the default of
        # admin.ShowFacets.ALLOW.
```